### PR TITLE
Add Backend Sandbox Script

### PR DESCRIPTION
**Pull request by theodormoroianu**
This PR adds a script for automatic bot testing, partially closing issue #9.

### CODE REVIEW

**Review by ConstantinDragancea — APPROVED** [review]
Seems legit

**Review by stelianchichirim — APPROVED** [review]
Nice

File: Engine/Grader/enginelib.hpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    ofstream out("bot_input");
+    out << state << '\n';
+    out.close();
+
+    string command = "/ia-sandbox/ia-sandbox " + executable_name +
+            " --stdin bot_input --stdout bot_output --stderr ignore " +
+            "-t " + to_string(int(time_limit_sec * 1000)) + "ms " +
+            "-o oneline > ia_sandbox_stdout";
+
```
Comment (felixp72) [inline Engine/Grader/enginelib.hpp:33]: Make a line for each of the arguments (e.g. stdin, stdout ....)

**Review by felixp72 — CHANGES_REQUESTED** [review]
Cool, just modify that command line :)

**Review by felixp72 — APPROVED** [review]
Neat

**Pull request merged**